deployments/examples/ocis_wopi/.env

```diff
@@ -45,7 +45,7 @@ OCIS=:ocis.yml
 OCIS_DOCKER_IMAGE=
 
 # The oCIS container version.
-# Defaults to "latest". This will point to the latest stable tag.
+# Defaults to "latest". Use the latest stable tag like 5.0.5.
```

Comment:
This change on my side was done by intention. ’latest’ will point to the latest stable tag. That is a change.

Reply:
> ’latest’ will point to the latest stable tag

Means that latest will point to 5.0.5 (currently)? If yes, I was not aware of that as docker hub confuses me with latest, 5.0.5 and their respective days old...

Reply:
That needs to be corrected.

Reply:
Q: as this PR is stable 5 only and there will be an own version for master or rolling releases, would it make problems accepting the change so we can proceed? I have to update docs for them in any case because of switching from wopiserver to collaboration...? So this would not be dependent where latest will point to in the future.

Reply:
let me please fix that first.